src/modules/transfer/views/ViewTransferCreate.vue

```diff
@@ -504,9 +533,9 @@ export default {
       }
     }else {
       if(tokenDivisibility== 0){
-        return Math.trunc((checkIsNaN(parseFloat(sendXPX.value.replace(/,/g, ''))) + parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value))
+        return Math.trunc((parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value))
       }else{
-        return Math.round((checkIsNaN(parseFloat(sendXPX.value.replace(/,/g, ''))) + parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value)*Math.pow(10,tokenDivisibility))/ Math.pow(10,tokenDivisibility)
+        return Math.round((parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value)*Math.pow(10,tokenDivisibility))/ Math.pow(10,tokenDivisibility)
       }
```

Comment:
why is this changed?

Reply:
the sendxpx value only needed to total up if it is non multisig. If it is multisig, the sendxpx value will only show for the primary(multisig) account and the total(Aggregate Tx Fee, Lock Fund Tx Fee and Lock Fund) for the signer.